prdoc/pr_2280.prdoc

```diff
@@ -48,7 +48,7 @@ doc:
           contextual data that is injected into the transaction extension logic. It is unused in
           instances migrated from `SignedExtension`.
         - Extensions now track the weight they consume during valdiation, preparation and
-          post-dispatch through the `TransactionExtensionBase::weight` function.
```

Comment:
PR 2280 was merged already, then reverted. Maybe we should leave this as is for posterity?